tasks/rollup.js

```diff
@@ -87,10 +87,10 @@ module.exports = function(grunt) {
         }
 
         return bundle.generate({
+          amd: { id: options.moduleId },
```

Comment:
Since you add the "amd.id" option. You could add "amd.define" too ?
For the option name, it could be "moduleDefine", but I can't speak for @chrisprice on this.